infra/build/functions/build_project.py

```diff
@@ -195,12 +195,18 @@ def get_env(fuzzing_language, build):
       'HOME': '/root',
       'OUT': build.out,
   }
+
   return list(sorted([f'{key}={value}' for key, value in env_dict.items()]))
 
 
 def get_compile_step(project, build, env, parallel):
   """Returns the GCB step for compiling |projects| fuzzers using |env|. The type
   of build is specified by |build|."""
+  set_git_repo_env = ''  #do nothing
+  if build.sanitizer == 'instrumentor':
+    set_git_repo_env = (
+        ' && export GITHUB_REPO=$(grep -P -o "\S+github.com\S+" /workspace/oss-fuzz/projects/'
```

Comment:
Can we do this in a less hacky way?

It's used by the helper.py script right?

We can instead modify the helper.py script to set this based on the `main_repo` property in project.yaml. e.g. https://github.com/google/oss-fuzz/blob/126cf1128ca9a632fb882097730333d39418741a/projects/alembic/project.yaml#L6. Let's do all this in another PR which upstreams the patches from fuzz introspector.

This is slightly different from how the existing patch does it: https://github.com/ossf/fuzz-introspector/blob/a49f0ca54103e6dc0177700d22a166a727683334/oss_fuzz_integration/oss-fuzz-patches.diff#L194 but it's less hacky.